gobblin-core/src/main/java/gobblin/source/extractor/extract/kafka/KafkaRecord.java

```diff
@@ -0,0 +1,87 @@
+package gobblin.source.extractor.extract.kafka;
+
+public class KafkaRecord {
+
+    private long offset;
```

Comment:
Please make these fields final, and remove the setter methods since they are not needed.
Also you can replace the getter methods with a @Getter annotation above line 3.